app/src/main/res/values/strings.xml

```diff
@@ -1138,4 +1138,5 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
 Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_shoulder_title">"Does this road have a shoulder?"</string>
     <string name="quest_fireHydrant_diameter_ref_title">What diameter is specified on the sign for this fire hydrant? (reference number: %s)</string>
+    <string name="quest_railway_electrification_system_title">Is this track section electrified?</string>
```

Comment:
```suggestion
    <string name="quest_railway_electrification_system_title">Is this railway track section electrified?</string>
```

Should we explicitly mention "railway" to avoid confusion with `highway=track`?

Reply:
Would probably be a good idea.